collection_templates/collections_torch.yml

```diff
@@ -0,0 +1,4 @@
+volume: "XX" # Replace XX with the volume number
```

Comment:
Naming convention needs double underscore for these files.

Reply:
@DonRichards - thanks for catching this. The double underscore didn't register with me visually. The files should be corrected now.